dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/BitBucketSourceCodeRepo.java

```diff
@@ -114,19 +110,13 @@ public String readFile(String fileName, String reference) {
             return null;
         }
 
-        if (content != null && !content.isEmpty()) {
```

Comment:
Will this work when content is null?

Reply:
I think this was an IntelliJ simplifification. 
Looks like line 114 always returns if the content is null, so this check was redundant.